docker-compose.test.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+services:
+  tests:
+    build:
+      context: .
+      target: ci
+      dockerfile: Dockerfile.test
+    environment:
+      - "MONGO_TEST_BASE_URI=mongodb://db:27017"
+    depends_on:
+      - db
+  db:
+    image: mongo
+    ports:
+      - "27017:27017"
```

Comment:
This isn't necessary unless you want to try accessing it directly from your host machine. The mongo image already "exposes" 27017.